customer/src/test/java/no/unit/nva/customer/get/GetCustomerHandlerTest.java

```diff
@@ -36,6 +18,25 @@
 import org.junit.jupiter.api.Test;
 import org.zalando.problem.Problem;
 
+import java.io.ByteArrayOutputStream;
```

Comment:
Are you using the NVA code style? A lot of stuff was shifted around here.

Reply:
Rurik hjalp meg med å sette opp rett CheckStyle, skal være i orden nå.

Men var det noe spesielt med akkurat denne linjen?